src/main/java/org/zalando/nakadi/validation/schema/diff/StringSchemaDiff.java

```diff
@@ -0,0 +1,23 @@
+package org.zalando.nakadi.validation.schema.diff;
+
+import org.everit.json.schema.StringSchema;
+import org.zalando.nakadi.domain.SchemaChange;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Stack;
+
+import static org.zalando.nakadi.domain.SchemaChange.Type.ATTRIBUTE_VALUE_CHANGED;
+
+class StringSchemaDiff extends SchemaDiff {
```

Comment:
What is the point to extend the SchemaDiff? I think it better to not extend it as you did for other classes because you use only static methods and there is no `is a` relation.